app/src/main/java/com/example/bigowlapp/activity/EditProfileActivity.java

```diff
@@ -64,23 +65,32 @@ private void onClickConfirmButton() {
             editUserLastName.requestFocus();
         }
 
+        if (!(fingerprintAuthRegistration.equalsIgnoreCase("NO")
+                || fingerprintAuthRegistration.equalsIgnoreCase("YES"))) {
+            editFingerprintAuthRegistration.setError("Please enter YES or NO and remove any empty space.");
+            editFingerprintAuthRegistration.requestFocus();
+        }
```

Comment:
I know it is not the usual way to do it, but in this case having the user to type "yes" or "no" makes it not necessary to add a confirmation display for their choice, since user has clearly written "yes" or "no". What user has to do is to enter at most a 3 letters word. Beside that, on edit profile, the user possibly has to enter names or phone number which are longer than the "yes" or "no". So I don't think it is something much to ask to user.
For the validity check, I agree. I can encapsulate that  in a method